src/saving/SaveList.tscn

```diff
@@ -44,26 +46,46 @@ text = "Loading..."
 align = 1
 
 [node name="ItemList" type="VBoxContainer" parent="MarginContainer/VBoxContainer"]
-margin_right = 1356.0
+margin_right = 1270.0
 
-[node name="DeleteConfirmDialog" type="ConfirmationDialog" parent="MarginContainer/VBoxContainer"]
-margin_top = 4.0
-margin_right = 1356.0
-margin_bottom = 80.0
+[node name="DeleteSaveConfirmDialog" type="ConfirmationDialog" parent="MarginContainer/VBoxContainer"]
+margin_top = 15.0
```

Comment:
@Kasterisk I unset all of the margins except the margin_top to 0 to get the dialogs to be the right size again (instead of always the screen width). Is that the right way to go?

Reply:
Probably not since the margins will just get automatically overridden by something else. I would instead disable the dialogs `fill` size flags

Reply:
I'll give that a try, but it seems that according to this diff, I didn't change the size flags, so it wasn't right before.

Reply:
Okay, I changed them.